ex4/index.js

```diff
@@ -0,0 +1,9 @@
+function numarVocale(sir) {
+       const vocale = șir.match(/[aeiou]/gi);
+       if (vocale === null) {
+        return 0;
+    }
+        return vocale.length;
+}
+const count = "Aici se gasesc cateva vOcalE";
+console.log(numarVocale(count)); 
```

Comment:
(opt) Se poate adauga un mesaj descriptiv inainte de output, de exemplu: `console.log("S-au gasit " + nr_vocale + " vocale in text");`